packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -279,6 +279,18 @@ const tests = {
         }
       `,
     },
+    {
+      code: normalizeIndent`
+      function MyComponent() {
```

Comment:
Neighbouring cases have a slightly bigger offset (+ 2 spaces). Might be worth aligning 🙂